logger_test.go

```diff
@@ -13,12 +13,7 @@ import (
 )
 
 func assertLogEntryContains(t *testing.T, logReader io.Reader, key string, expectedValue interface{}) {
-	t.Helper()
```

Comment:
Nice refactoring of the `decodeLogToMap`.
But please leave the `t.Helper` in each of the helper-funcs, as moving them to the `decodeLogToMap` func reduces readability in test-errors.